<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VoxelProcessorClass</class>
 <widget class="QMainWindow" name="VoxelProcessorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VoxelProcessor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tab_mainWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_voxelScene">
       <attribute name="title">
        <string>Scene</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSplitter" name="spl_viewerAll">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QSplitter" name="spl_managerandViewer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QSplitter" name="spl_managerandAttribute">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <widget class="VoxelSceneManagerWidget" name="wid_voxelScene" native="true">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>200</height>
              </size>
             </property>
            </widget>
            <widget class="VoxelAttributeWidget" name="wid_objectAttribute" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>200</height>
              </size>
             </property>
            </widget>
           </widget>
           <widget class="VoxelViewerOGLWidget" name="ogl_voxelViewer">
            <property name="minimumSize">
             <size>
              <width>400</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </widget>
          <widget class="QScrollArea" name="sa_voxelTools">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>83</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOn</enum>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="sa_voxelToolsContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>64</width>
              <height>512</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="pb_view">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>64</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>64</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>View</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pb_translate">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>64</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>64</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Translate</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pb_rotate">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>64</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>64</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Rotate</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pb_scale">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>64</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>64</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Scale</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pb_df">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>64</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>64</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>DF</string>
               </property>
               <property name="autoDefault">
                <bool>false</bool>
               </property>
               <property name="default">
                <bool>false</bool>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pb_thickness">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>64</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>64</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Thickness</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pb_boolean">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>64</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>64</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Boolean</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pb_slice">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>64</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>64</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Slice</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_voxelEditor">
       <attribute name="title">
        <string>Editor</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSplitter" name="spl_editWidgetAll">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QSplitter" name="spl_listandViewer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QSplitter" name="spl_listandAttribute">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <widget class="VoxelDataListManagerWidget" name="wid_dataList" native="true">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>200</height>
              </size>
             </property>
            </widget>
            <widget class="VoxelAttributeWidget" name="wid_dataAttribute" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>200</height>
              </size>
             </property>
            </widget>
           </widget>
           <widget class="VoxelEditorOGLWidget" name="ogl_editViewer">
            <property name="minimumSize">
             <size>
              <width>400</width>
              <height>0</height>
             </size>
            </property>
            <property name="mouseTracking">
             <bool>true</bool>
            </property>
           </widget>
          </widget>
          <widget class="QScrollArea" name="sa_editTools">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>83</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOn</enum>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="sa_editToolsContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>64</width>
              <height>502</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="pb_selection">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>64</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>64</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Selection</string>
               </property>
               <property name="autoDefault">
                <bool>false</bool>
               </property>
               <property name="default">
                <bool>false</bool>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pb_control">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>64</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>64</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Control</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pb_freeform">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>64</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>64</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>FreeForm</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pb_smoothing">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>64</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>64</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Smoothing</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pb_connectingcomponent">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>64</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>64</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Connecting
Component</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pb_7">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>64</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>64</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>7</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pb_8">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>64</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>64</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>8</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>795</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VoxelViewerOGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>voxelvieweroglwidget.hpp</header>
  </customwidget>
  <customwidget>
   <class>VoxelSceneManagerWidget</class>
   <extends>QWidget</extends>
   <header>voxelscenemanagerwidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VoxelEditorOGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>voxeleditoroglwidget.hpp</header>
  </customwidget>
  <customwidget>
   <class>VoxelDataListManagerWidget</class>
   <extends>QWidget</extends>
   <header>voxeldatalistmanagerwidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VoxelAttributeWidget</class>
   <extends>QWidget</extends>
   <header>voxelattributewidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
